lib/integer_helper/integer_parser.ex

```diff
@@ -0,0 +1,27 @@
+defmodule ElixirTools.IntegerHelper do
```

Comment:
name of the file `lib/integer_helper/integer_parser.ex` -> `lib/integer_helper.ex` ?